### DISCUSSION TITLE
Quotation marks missing from Italic

### DISCUSSION THREAD

**Discussion by carbeck** [psb1558/Junicode-font#101] [General]
Dear Peter, I've noticed that the non-English quotation marks (U+201A, U+201B, U+201E, U+201F) are missing from the Italic so far. I don't know if that's merely an oversight or whether you purposefully put those glyphs on hold for working on them later, given that they're present in the Regular.

**Comment by psb1558** [comment]
Thanks for the query, Carsten. The italic face is less finished than the roman, with (currently) 191 glyphs present in the source but needing work before they can be included in the generated  font. Those four quotation marks (and 23 others!) are among the 191. I expect the italic to catch up to the roman around the middle of this year.
